clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -24,17 +21,29 @@ import { useConsentServed } from "../../lib/hooks";
 import { updateCookieFromNoticePreferences } from "../../lib/cookie";
 import PrivacyPolicyLink from "../PrivacyPolicyLink";
 import { dispatchFidesEvent } from "../../lib/events";
+import { resolveConsentValue } from "../../lib/consent-value";
+import { getConsentContext } from "../../lib/consent-context";
+import { transformConsentToFidesUserPreference } from "../../lib/shared-consent-utils";
 
 const NoticeOverlay: FunctionComponent<OverlayProps> = ({
   experience,
   options,
   fidesRegionString,
   cookie,
 }) => {
-  const initialEnabledNoticeKeys = useMemo(
-    () => Object.keys(cookie.consent).filter((key) => cookie.consent[key]),
-    [cookie.consent]
-  );
+  const consentContext = useMemo(() => getConsentContext(), []);
+  const initialEnabledNoticeKeys = useMemo(() => {
+    if (experience.privacy_notices) {
+      return experience.privacy_notices?.map((notice) => {
```

Comment:
no need for the `?` here since you have the `if` above
```suggestion
      return experience.privacy_notices.map((notice) => {
```